runtime/statemine/src/weights/pallet_balances.rs

```diff
@@ -19,10 +34,12 @@
 // --extrinsic
 // *
 // --steps
-// 20
+// 50
 // --repeat
-// 10
+// 20
```

Comment:
You can make these one line by putting `--steps=50` instead of `--steps 50`. Looks a little nicer

Reply:
thanks, I made the changes to benchmarks.sh script, will probably have to run the weights one more time anyways.